plugins/texteditor.koplugin/main.lua

```diff
@@ -179,6 +182,29 @@ Export text to QR code, that can be scanned, for example, by a phone.]]),
                     callback = function()
                         self.qr_code_export = not self.qr_code_export
                     end,
+                    separator = true,
+                },
+                {
+                    text = _("Clean text editor history"),
+                    enabled_func = function()
+                        return #self.history > 0
+                    end,
+                    keep_menu_open = true,
+                    callback = function(touchmenu_instance)
+                        UIManager:show(ConfirmBox:new{
+                            text = _("Clean text editor history?"),
+                            ok_text = _("Clean"),
```

Comment:
"Clean" or "Clear" ?

Reply:
https://github.com/koreader/koreader/blob/2c1cae64b61ba20abd9f7f5d6dac27abdadde35e/frontend/apps/reader/modules/readerdictionary.lua#L289
https://github.com/koreader/koreader/blob/2c1cae64b61ba20abd9f7f5d6dac27abdadde35e/frontend/apps/reader/modules/readerwikipedia.lua#L301

Reply:
That's @Frenzie original wording, so it must be right :)
(I get the feeling that when "Cleaning", we'd expect to get valuable stuff remaining/ordered, unlike "Clearing" which I expect to make it empty, as it's what it does :)

Reply:
I think I went with what was already used elsewhere at the time but clean seems fine to me, yes.